learning-outcomes.md

```diff
@@ -0,0 +1,59 @@
+# Week 1 Learning Outcomes
+
+## Teamwork & Toolkit
+
+### Pair programming
+
+- [ ] **We can use pair programming techniques to work together**
+- [ ] We can work on one computer as a pair
+- [ ] We can use four different paired programming techniques
+- [ ] We can distribute commits evenly among team members
+
+---
+
+### Version control
+
+- [ ] **We can use Git and GitHub as version control for our code**
+- [ ] We can initialise a Git repository so that we can track changes to our code over time
+- [ ] We can add / stage files so that we can include these updates in our next commit
+- [ ] We can write descriptive commit messages so that our commit history tells a story
+- [ ] We can amend commit messages so that we can correct typos/mistakes before we push to GitHub
+- [ ] We can co-author commit messages so that our commit history reflects everyone’s contributions
+- [ ] We can merge master to our feature branch so that we can resolve any merge conflicts locally
+- [ ] We can resolve merge conflicts to maintain a working codebase
+- [ ] We can push and pull branches from a remote to work on different tasks
+- [ ] We can create a pull request on GitHub to request feedback from our teammates
+- [ ] We can review other people’s pull requests to provide feedback on their code
+- [ ] We can delete branches that have been merged to keep our repo tidy
+
+---
+
+### Accessibility
+
+- [ ] **We can make a website that is accessible by people other than ourselves**
+- [ ] We can structure a site using semantic HTML to aid accessibility
+- [ ] We can add appropriate attributes to HTML elements to make them screen reader friendly
+- [ ] We can add ARIA labels (where necessary) to make our semantic HTML more accessible
+- [ ] We can run a dev tools accessibility audit to check our website’s accessibility
+- [ ] We can use a screen reader to understand how people with visual impairments may experience our website
+- [ ] We can design a UI without relying solely on colour, so that we don't exclude colour-blind users
```

Comment:
These two (line 39 and 40) could be combined

- we can choose colours and contrast ratios in our UI to make our design inclusive for colour-blind and visually impaired users

Reply:
If we're going to mention any specific access requirements I would quite like to keep these as independent points, as they're not really related (other than being colour).

I do wonder if it makes sense to have a random selection of a11y stuff in here though. There's a big checklist of stuff to do/not do, and it feels weird to pick out just a few of them (and might mislead students into thinking these are more important). Maybe we should have a more generic a11y LO?